idea_town/frontend/static-src/app/templates/not-found-page.js

```diff
@@ -0,0 +1,6 @@
+export default `
+  <section class="page" data-hook="not-found-page">
+    <h1>404 wut</h1>
+  </section>`
```

Comment:
Filed #87 to track whether we can get our Travis container to also run some Node+Gulp tasks.
I haven't played too much with multi-language Travis projects yet. /cc @nchapman (whom I think did this with "Find My Device" w/ Go+Node)